flutter/lib/ui/geometry.dart

```diff
@@ -837,39 +982,47 @@ class RRect {
   }
 
   /// The point halfway between the left and right and the top and bottom edges of this rectangle.
-  Point get center => new Point(left + width / 2.0, top + height / 2.0);
+  Point get center => new Point(left + width / 2, top + height / 2);
```

Comment:
Not sure I understand. How is it more consistent to make it an int when everything else is double?